src/routes/safe/components/Apps/confirmTransactions.js

```diff
@@ -8,6 +9,8 @@ import Heading from '~/components/layout/Heading'
 import Img from '~/components/layout/Img'
 import { getEthAsToken } from '~/logic/tokens/utils/tokenHelpers'
 
+const humanReadableBalance = (balance, decimals) => BigNumber(balance).times(`1e-${decimals}`).toFixed()
```

Comment:
I think we already have sort of function like this, as far I remember @fernandomg has worked on exactly this conversion, maybe he already have an utility function like this

Reply:
agreed, can we create a separate ticket to refactor this and all other places that are replicating the exact same code? I took it from another place.

Reply:
yes sure, feel free to create a new ticekt

Reply:
https://github.com/gnosis/safe-react/issues/877